web/org.eclipse.xtext.web.example.jetty/test-files/example.statemachine

```diff
@@ -12,4 +12,8 @@ end
 
 state State2
 	if x == false and y == true goto State1
-end
\ No newline at end of file
+end 
```

Comment:
I'm not sure I understand what this change has to do with this PR (Xtend Web Editor) - perhaps you would like to undo to remove this changed file from this PR?